proto/api/v1/api.proto

```diff
@@ -120,11 +112,10 @@ message Processor {
   // -- immutable fields -------------------------------------------------------
 
   string name = 4 [(google.api.field_behavior) = IMMUTABLE];
-  Type type = 5 [(google.api.field_behavior) = IMMUTABLE];
-  Parent parent = 6 [(google.api.field_behavior) = IMMUTABLE];
+  Parent parent = 5 [(google.api.field_behavior) = IMMUTABLE];
```

Comment:
IMO we shouldn't change existing field numbers as that would be a breaking change. I would also prefer to have nicely numbered fields of course, but it's not worth it if it introduces a breaking change. I'd say we can go through the proto files before releasing version 1 of Conduit and clean them up since we would be bumping a major version at that point.

Reply:
Sure, I can revert. Question before that: do we care about breaking changes at this point?

Reply:
Not that much really, but it's an easy one to avoid.